src/components/CommentComposer/CommentComposerHeader.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+import {css} from 'glamor'
+import {MdCheck, MdEdit} from 'react-icons/lib/md'
+import colors from '../../theme/colors'
+import {sansSerifMedium16, sansSerifRegular14} from '../Typography/styles'
+
+const styles = {
+  root: css({
+    display: 'flex',
+    alignItems: 'center',
+    background: colors.whiteSmoke,
```

Comment:
Just for the record:
As discussed, these colors need to replaced by functional names once the color scheme is better defined: